test/helpers/db.js

```diff
@@ -27,12 +28,22 @@ const migrationTables = [
 // Only run migrations once during tests
 let init = false
 exports.init = function * () {
+  // TODO: Create dropTable fn and run before createTable
```

Comment:
I don't think this comment is necessary anymore (I had it in my branch as a reminder)